core/version/gear.ex

```diff
@@ -113,20 +115,23 @@ defmodule AntikytheraCore.Version.Gear do
   @typep gear_dependencies :: MapSet.t()
   @typep gear_and_deps_pair :: {GearName.t(), gear_dependencies}
 
-  defun install_gears_at_startup(gear_names :: v[[GearName.t()]]) :: :ok do
+  defun install_gears_at_startup(gear_names :: v[[GearName.t()]]) :: :ok | :error do
     if Antikythera.Env.running_with_release?() do
       do_install_gears_at_startup(gear_names)
     else
       :ok
     end
   end
 
-  defunp do_install_gears_at_startup(gear_names :: v[[GearName.t()]]) :: :ok do
+  defunpt do_install_gears_at_startup(gear_names :: v[[GearName.t()]]) :: :ok | :error do
+    # Tests need `__MODULE__.` to mock these functions.
     gear_and_deps_pairs =
-      Enum.map(gear_names, fn g -> {g, gear_dependencies_from_app_file(g, gear_names)} end)
+      Enum.map(gear_names, fn g ->
+        {g, __MODULE__.gear_dependencies_from_app_file(g, gear_names)}
+      end)
 
-    pairs_not_installed =
-      install_gears_whose_deps_met(gear_and_deps_pairs, MapSet.new(), fn gear_name ->
+    {pairs_not_installed, num_installed} =
```

Comment:
I think the value of num_installed can be calculated as follows:
```elixir
num_installed = length(gear_names) - length(pairs_not_installed)
```

Reply:
Fixed at
https://github.com/access-company/antikythera/pull/152/commits/a97aef3a7f5b8e49e6b083779fef95fb9288372c